src/lib/NumberIncrementer/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FunctionComponent, useState, useCallback } from 'react'
+import { createCounterSubtractFactory, createCounterAddFactory } from './factory.counter'
+import clsx from 'clsx'
+
+import styles from './numberIncrementer.module.scss'
+
+interface Props {
+  className?: string
+  [rest: string]: unknown // ...rest property
+}
+
+const NumberIncrementer: FunctionComponent<Props> = ({ className, ...rest }) => {
+  const [number, setNumber] = useState(1)
```

Comment:
pass a prop like `defaultValue` that sets this. Also make it controlled by passing in a value prop

Reply:
And actually, I would use the `useControlled` hook. Check out the `Rating` component to see how it works